package.json

```diff
@@ -5,7 +5,7 @@
   "main": "lib/index.js",
   "scripts": {
     "build": "flow-remove-types -d lib/ src/",
-    "test": "npm run build && mocha",
+    "test": "npm run build && mocha -t 5000",
```

Comment:
This seems bad, can you debug why the testcases are running so long by building a minimal testcase?